src/opcode.def

```diff
@@ -48,16 +48,16 @@ WABT_OPCODE(___,  ___,  ___,  ___,  0,  0,    0x09, Rethrow, "rethrow", "")
 WABT_OPCODE(___,  ___,  ___,  ___,  0,  0,    0x0a, BrOnExn, "br_on_exn", "")
 WABT_OPCODE(___,  ___,  ___,  ___,  0,  0,    0x0b, End, "end", "")
 WABT_OPCODE(___,  ___,  ___,  ___,  0,  0,    0x0c, Br, "br", "")
-WABT_OPCODE(___,  ___,  ___,  ___,  0,  0,    0x0d, BrIf, "br_if", "")
```

Comment:
Remind me what these changes do again?  Would these have been useful outside of this change?  (I guess I'm saying can this be separated?)

Reply:
This makes it easier to handle tracing; these types can be extracted directly so the values can properly be displayed in the interpreter (e.g. what type to use for a given `Value`)